.ci/packaging.groovy

```diff
@@ -38,8 +39,29 @@ pipeline {
     upstream("Beats/beats/${ env.JOB_BASE_NAME.startsWith('PR-') ? 'none' : env.JOB_BASE_NAME }")
   }
   parameters {
-    booleanParam(name: 'macos', defaultValue: false, description: 'Allow macOS stages.')
-    booleanParam(name: 'linux', defaultValue: true, description: 'Allow linux stages.')
+    booleanParam(name: 'linux', defaultValue: true, description: 'Build linux binaries.')
+    booleanParam(name: 'macos', defaultValue: true, description: 'Build macOS binaries.')
+    booleanParam(name: 'windows', defaultValue: true, description: 'Build windows binaries.')
+
+    booleanParam(name: 'i386_amd64', defaultValue: true, description: 'Build i386/amd64 binaries.')
+    booleanParam(name: 'arm', defaultValue: true, description: 'Build ARM binaries.')
+    booleanParam(name: 'ppc_mips_x390s', defaultValue: true, description: 'Build PPC/MIPS/x390s binaries.')
+
+    booleanParam(name: 'auditbeat', defaultValue: true, description: 'Build auditbeat binaries.')
+    booleanParam(name: 'filebeat', defaultValue: true, description: 'Build filebeat binaries.')
+    booleanParam(name: 'heartbeat', defaultValue: true, description: 'Build heartbeat binaries.')
+    booleanParam(name: 'journalbeat', defaultValue: true, description: 'Build journalbeat binaries.')
+    booleanParam(name: 'metricbeat', defaultValue: true, description: 'Build metricbeat binaries.')
+    booleanParam(name: 'packetbeat', defaultValue: true, description: 'Build packetbeat binaries.')
+    booleanParam(name: 'winlogbeat', defaultValue: true, description: 'Build winlogbeat binaries.')
+    booleanParam(name: 'elastic_agent', defaultValue: true, description: 'Build elastic-agent binaries.')
+    booleanParam(name: 'dockerlogbeat', defaultValue: true, description: 'Build dockerlogbeat binaries.')
+    booleanParam(name: 'functionbeat', defaultValue: true, description: 'Build functionbeat binaries.')
+
+    booleanParam(name: 'sign_on_macos', defaultValue: false, description: 'Sign macOS binaries.')
+    booleanParam(name: 'archive_on_gcp', defaultValue: true, description: 'Enable/Disable the archive of binaries on GCP.')
+    booleanParam(name: 'push_docker', defaultValue: true, description: 'Enable/Disable the push of the Docker images.')
+    booleanParam(name: 'dry_run', defaultValue: false, description: 'Execute the pipeline without generate packages.')
```

Comment:
Is it required to convert those params to env variables to workaround JENKINS-41929?

Reply:
no needed, the pipeline uses the parameters not the environment variables